setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
```

Comment:
me too, but bdist_wheel is the documented option https://bitbucket.org/pypa/wheel/src/f746a01eab7a673cb628a52f99c129d0d5741bcf/setup.cfg?at=default#cl-7